snapcraft.yaml

```diff
@@ -12,6 +11,13 @@ apps:
     daemon: simple
     plugs: [listener]
 
+  # PHP-FPM daemon
```

Comment:
That's the daemon responsible for keeping PHP-FPM alive and ready to process requests